File: src/main/scala/_01_helloWorld/userInput.scala
package gay.jons
package _01_helloWorld

import scala.io.StdIn.readLine
import io.AnsiColor._

@main def helloInteractive(): Unit =
  println("Please enter your name")
  val name: String = readLine()
  println(s"Hello, ${BOLD}${MAGENTA}${name}!${RESET}")
  println("I'm glad you came")

File: src/main/java/gay/jons/_01_helloWorld/UserInput.java
package gay.jons._01_helloWorld;

import java.util.Scanner;

public class UserInput {
    public static void main(String[] args) {
        System.out.println("Please enter your name");

        Scanner sc = new Scanner(System.in);
        System.out.println("Hello, " + sc.next() + "!");
    }
}
